### ISSUE TITLE
[Req] Combine Wizard - adding to the offer

### ISSUE THREAD

**Issue by teessl** [danny0838/firefox-scrapbook#294]
Hey. 
I've been using ScrapBook for years and only recently came to like combining saved pages.
I have to go to Tools and select there Combine Wizard...

More intuitive would be, I use the sidebar, to add to the menu on the marked pages:
Copy Page Info
Tools >
Delete

To tools - Combine or or right in the main menu add Combine pages (wizard, fast combine save here).
Also of interest would be the ability to save the resulting file to a folder of the original. 
It's always root and for many directories you need to search.

"After completing combine, remove original pages."
I'm missing the option to check the directory from which the combined files come from. Or a list of frequently used directories.
It's a cool thing, but it takes an unnecessary amount of time if one uses it more often.
Thank you very much for your help!

**Comment by danny0838** [comment]
The "delete original page after combine" option is available in the preview page.

**Comment by teessl** [comment]
Yes I know. Useful thing.
That's why I wrote that there could possibly be an option to save the combined page to the directory where the first page was from. It's not an optimal solution, just an option.
I have a lot of directories, a lot of subdirectories, 99.9% of the combined pages I store in the same directory.
I can also imagine someone combining sites from different directories because of a topic, a job, a collaboration, but that's not my case. That is not essential.

For me it would be very helpful to be able to directly combine the marked pages from the sidebar possibly with the option to save directly here, which doesn't have to be, it would be enough to put them directly into the combine wizard where the base save option might not be root, but the directory of the first page.

To clarify what the problem is. When I have a full-screen browser, or a larger part of it, I have to position the Combine wizard so that I can move the selected pages to the "Add items by drag-and-drop" box, i.e. make the browser window smaller, move it. A little impractical to combine 2-3 pages, several times a day.

**Comment by teessl** [comment]
I don't want to open a new request, I just figured one thing out not too long ago.
When I save subpages "in-depth Save" Depth to follow links: 1
Most of the time it works great. Sometimes it doesn't and then a combination of sites is better, but I found that I can't find the original web address subpages anywhere, except on the main page.
The original address could be in SiteMap as the second line. 

Usage:
Recently I had an idea to save an order from an shop with subpages (I don't know why I didn't think of it before).
On one store it works great, on another not at all, some error 503. When I viewed the order a week later and clicked on the subpage, it opened the one saved by ScapBook X. That's fine, that's what I wanted. But I can't find the original web address anywhere to get that subpage to open on the online site. So that I can order the goods again or check for changes. I had to put the goods in search.

Even though I'm on a subpage, down in the yellow bar is still the address of the main page https:// It would be great if there was the original address of the subpage.  I click and it loads the online version.
Thank you very much!

**Comment by danny0838** [comment]
ScrapBook X is a very old project. Browsers supporting it are too old to support most modern techniques, making implementing any good new feature difficult or impossible.

As there is already [WebScrapBook](https://github.com/danny0838/webscrapbook) as the WebExtension port, we will keep only minimum maintenance for ScrapBook X. Regretfully, the requests you have raised require very large code rework, and thus it's very, very unlikely that they'll be implemented, although we won't clearly say no or impossible.

Due to the wide diversity of modern web techniques, combining page is technically very difficult and error-prone, and even the current implementation has lots of potential issues, such as breakage of scripts and styles and losing in-depth saved pages, as you probably have discovered.

Generally speaking, combining page is deprecated. Instead, organize them using a folder item or use in-depth capture or [merge capture](https://github.com/danny0838/webscrapbook/wiki/Basic#merge-capture) (introduced in WebScrapBook).

**Comment by teessl** [comment]
Thank you for your reply. 
Is this also about giving the original web address https://github.com/danny0838/firefox-scrapbook/issues/294#issuecomment-848392226  or is this already in WebScrapBook?

It's nothing extremely important, ScrapBook is the best. I also use SingleFile, mhtml, I've tried others, but ScrapBook is the best, with universal use.
Thank you for your time, work, maintenance and development!

**Comment by danny0838** [comment]
Yes and no. A scrapbook item is not designed to track source of all captured pages. However, the source URL of all in-depth captured pages are actually recorded, which is at `index.json` for WebScrapBook and `sb-url2name.txt` for ScrapBook, and can be used for back-tracing on demand, but it's mainly for using for the application internally and not designed for the user to use directly.

WebScrapBook also records the source URL as an attribute in the root HTML element for all in-depth captured pages. You can trace it via the page source code.

**Comment by teessl** [comment]
> However, the source URL of all in-depth captured pages are actually recorded, which is at index.json for WebScrapBook and sb-url2name.txt for ScrapBook, and can be used for back-tracing on demand

Thank you, I looked it up. It's sufficient for my occasional use. ScrapBook is an awesome extension! Thank you again for your help, great work!
